recipes/la_forge/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "la_forge" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0b457003af2bcfe49b3320708520eb9e93a7b882f6b318b388f3107b9f630eee
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - numpy >=1.16
+    - scipy >=1.0.0
+    - matplotlib-base
+    - astropy >=3.0
+    - corner
+    - six
```

Comment:
I cannot find this upstream.

Reply:
I see these in setup.py. Although matplotlib instead of matplotlib-base. Is there somewhere else that I should put them?

Reply:
I have now double checked that they are there (at least on the main branch). Again, we'll get it on PyPI tomorrow.

Reply:
Ahh yes, you‘re right. I was looking at `main` - my bad.